components/ResultListPage/PlayAgainButton/index.js

```diff
@@ -12,7 +12,7 @@ export default function PlayAgainButton({
   const onHandleCurrentRound = () => {
     if (currentRound < 3) {
       setCurrentRound((prevCurrentRound) => prevCurrentRound + 1);
-      router.push("/");
+      router.push("/Quiz");
```

Comment:
The pages (and thus the URL route) is normally written in lowercase.
```suggestion
      router.push("/quiz");
```

Reply:
Thanks Ralf I already had the error and didn't want to do it again :D